docs/telemetry.md

```diff
@@ -1,55 +1,273 @@
+
+Your privacy is protected. Percona doesn't collect any personal information about you. All collected data is anonymous, meaning it can't be traced back to any individual user. To learn more about how Percona handles your data, read the [Percona Privacy statement](https://www.percona.com/privacy-policy).
+
+Currently, telemetry is added only to the Percona packages for both basic and Pro builds and to Docker images. 
 
 ## What information is collected
 
-Currently, telemetry is added only to the Percona packages and Docker images. It collects only information about the installation environment. Future releases may add additional telemetry metrics.
+Percona collects information using the following components:
+
+* The Telemetry subsystem collects the necessary metrics directly from the database and stores them in a Metrics file.
+
+* The Metrics file stores the metrics and is a standalone file located on the database host's file system.
+
+* The Telemetry Agent is an independent process running on your database host's operating system and carries out the following tasks:
+
+    * Collects OS-level metrics
+
+    * Reads the Metrics file, adds the OS-level metrics
+
+    * Sends the full set of metrics to the Percona Platform
+
+The telemetry also uses the Percona Platform with the following components:
+
+* Telemetry Service - offers an API endpoint for sending telemetry. The service handles incoming requests. This service saves the data into Telemetry Storage.
+
+* Telemetry Storage - stores all telemetry data for the long term.
+
+### Telemetry subsystem
+
+Telemetry subsystem extends the functionality of the database. It can be implemented in the following ways, depending on the database:
+
+* Modification to the source code
+
+* Modular components added to the database
+
+* Self-contained extensions
+
+Percona Server for MongoDB has built-in Telemetry subsystem implemented separately for `mongod` and `mongos` instances. It is enabled by default during the initial database deployment.
+
+Telemetry subsystem does not collect the following information:
+
+* database names
+
+* user names or credentials
+
+* any user-entered value
+
+Telemetry subsystem sends metrics gathered from the database instance to the Metrics file. This information is collected every 24 hours. Each connection creates a new Metrics file. When the file date exceeds seven days, the component removes the outdated files before creating the new file.
+
+The component creates a file in the local file system using a timestamp as the name with a `.json` extension.
+
+### Metrics file
+
+Percona stores the Metrics file in one of the following directories on the local file system. The location depends on the product.
+
+* Telemetry root path - `/usr/local/percona/telemetry`
+
+* Percona Server for MongoDB has two root paths, since telemetry subsystem is enabled both for the `mongod` and `mongos` instances. The paths are the following:  
```

Comment:
```suggestion
* Percona Server for MongoDB has two root paths since the telemetry subsystem is enabled both for the `mongod` and `mongos` instances. The paths are the following:  
```